frame/support/src/traits.rs

```diff
@@ -1664,6 +1664,28 @@ pub trait Instance: 'static {
     const PREFIX: &'static str ;
 }
 
+pub trait DustCollector<AccountId> {
+	fn is_collectable(who: &AccountId) -> bool;
```

Comment:
Am I wrong to think `is_dust()` is a better name for this? collecting doesn't quite imply that it is being killed.